codespell_lib/data/dictionary.txt

```diff
@@ -18999,9 +19033,13 @@ intercontinential->intercontinental
 intercontinetal->intercontinental
 interdependant->interdependent
 interecptor->interceptor
+interect->intersect, interact,
+interected->intersected, interacted,
 interecting->intersecting, interacting,
 interection->intersection, interaction,
+interectioned->instersectioned
```

Comment:
Suggested word is a misspelling. At the same time `intersectioned` is also a misspell. So you might want to just suggest `intersection`. Anyway, there is only 1 hit.
```text
interectioned->intersection
```
https://github.com/search?q=interectioned&type=code (1)